benchopt/runner.py

```diff
@@ -338,7 +345,10 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
     else:
         save_file = output_dir / f"{output_name}.csv"
         save_file = uniquify_results(save_file)
-    df.to_csv(save_file)
+    with open(save_file, 'a') as f:
+        f.write(f'#tag: {tag}\n')
```

Comment:
I think a more explicit name would be nice:
```suggestion
        f.write(f'# benchmark-git-tag: {tag}\n')
```